src/sci/core.cljc

```diff
@@ -303,7 +303,7 @@
             (let [m (:meta var)]
               (assoc ns-map var-name
                      (new-var (symbol var-name) (:val var)
```

Comment:
Maybe we can re-use the first `(symbol ..)` call

Reply:
yeah, I can pull that up into the `let` - I was a little bit curious, because as far as I can tell `var-name` should always be a symbol already, right?

Reply:
Maybe in the CLJS case it was a name.

Reply:
I'm really not sure, we can remove the symbol call and see what happens.

Reply:
I'm working on beefing up the test a little bit to ensure that they're always symbols, and I'll push a new commit shortly.